### DISCUSSION TITLE
Mr. Spock plotted on the Tek 4015

### DISCUSSION THREAD

**Discussion by nbreeden2** [thehighnibble/thehighnibble.github.io#32] [Show and tell]
Not really something I made but I've plotted Mr. Spock on the Tek 4015 and wanted to share.
![spock](https://user-images.githubusercontent.com/37715892/223542554-1f5a718f-50d8-4b15-9afc-4bf8268b616e.JPG)
-Neil

**Comment by villaromba** [comment]
I too plotted this via the Imsai8080 Tek window, link below plus the means to generate a .plt file from your own pic!!!, although I've not tried it yet.
https://github.com/rricharz/make-tek-image

**Comment by nbreeden2** [comment]
I've modified the c code a bit to brighten up the picture.
![neil-plot-1](https://user-images.githubusercontent.com/37715892/223788036-94b0c872-632d-42de-988f-9b6be7222957.JPG)

**Reply by villaromba** [reply]
Ah! That looks much improved, I will try and have a go with the prog. over the week-end.

**Comment by nbreeden2** [comment]
Updated to generate pixies 1:1 between the original jpg file and the Tek terminal. Being able to see the hair on my arms is really amazing.
![neil-plot-3](https://user-images.githubusercontent.com/37715892/223829256-27d8c2ac-56bb-456f-9c9f-d966cea96b22.JPG)

**Comment by villaromba** [comment]
![IMAGE](https://user-images.githubusercontent.com/52318473/224054061-e94d5b2a-33f7-4dd4-aafe-616961fd2a7e.jpg)
 Well I managed to run it all successfully !!!!
Colin

**Reply by nbreeden2** [reply]
Very nice!
